graphql/src/schema/api.rs

```diff
@@ -25,6 +25,7 @@ pub enum APISchemaError {
 }
 
 const BLOCK_HEIGHT: &str = "Block_height";
+const CHANGE_BLOCK_FILTER_NAME: &str = "BlockChangedFilter";
```

Comment:
@lutter I saw that we don't have a clear naming convention standard for the built GraphQL schema, I used PascalCase because that's what we are using for inputs.

Reply:
You are right - it seems that most other type names use PascalCase; I might have screwed that up in the past when I added `Block_height` instead of `BlockHeight`; I think using that name is good and continues the more prevalent pattern.